src/components/CardList/CardList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+import { CardInfoContext } from 'CardInfoContextProvider';
+
+import Card from 'components/common/Card/Card';
+
+function CardList() {
+  const { state } = useContext(CardInfoContext);
+  const { card, cards } = state;
+
+  return (
+    <ul>
+      {cards.length
+        ? cards.map((item) => (
+            <li key={item.nickname}>
+              <Card isEmpty={false} cardInfo={item} />
+              <div className="nickname-text">{item.nickname}</div>
+            </li>
+          ))
+        : ''}
```

Comment:
`false` 일 때 따로 렌더링할 게 없다면 `&&` 연산자를 보통 많이 사용합니다!
```suggestion
      {cards.length > 0 && cards.map((item) => (
            <li key={item.nickname}>
              <Card isEmpty={false} cardInfo={item} />
              <div className="nickname-text">{item.nickname}</div>
            </li>
          ))}
```